website/animeketchup/ak_api/urls.py

```diff
@@ -3,8 +3,9 @@
 
 urlpatterns = [
     path("", views.landing_page, name='landing_page'),
-    path("anime/", views.list_anime, name='list_anime'),
+    path("api/anime/", views.json_anime_list, name='json_anime_list'),
```

Comment:
Renamed to improve simplicity and to avoid confusion. 

Previously you would go to `http://127.0.0.1/anime/` and it would populate the page with the JSON content of all of the anime which has been created and POST to the API.